sdk/documenttranslation/Azure.AI.DocumentTranslation/README.md

```diff
@@ -1,19 +1,488 @@
 # Azure Cognitive Services Document Translation client library for .NET
+Document Translation is a cloud-based feature of the Azure Translator service and is part of the Azure Cognitive Service family of REST APIs. The Document Translation API translates documents to and from 90 languages and dialects while preserving document structure and data format.
```

Comment:
Python also created a [PR](https://github.com/Azure/azure-sdk-for-python/pull/17412) for the Readme, and we can copy some of the content in here too (where it applies) so Readme's can be as close as possible